inference-engine/include/ie_compound_blob.h

```diff
@@ -137,6 +127,20 @@ class INFERENCE_ENGINE_API_CLASS(CompoundBlob): public Blob {
      * @brief Returns nullptr as CompoundBlob is not allocator-based
      */
     void* getHandle() const noexcept override;
+
+    /**
+     * @brief Clones this container. Cloned container references the same blobs as this one.
+     *
+     * @return Pointer to the cloned object
+     */
+    Blob* clone() const override;
```

Comment:
Is copy assignment operator available along with `clone`?

Reply:
must be a dup of https://github.com/openvinotoolkit/openvino/pull/879#discussion_r440829811